package.json

```diff
@@ -9,8 +9,8 @@
     "url": "https://github.com/angular/angular.js.git"
   },
   "engines": {
-    "node": "<5",
-    "npm": "~2.5"
+    "node": "^6.9.0",
```

Comment:
Maybe `^6.9.1`? 6.9.1 was released a day after 6.9.0 and contained critical fixes.